includes/dashboard/viewUsers.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/* 
+ * this is to view users category wise
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+
+
+
+echo "<p>To vied the users groupwise, select the group and click 'view selected group'</p><br><br>";
+
+
+$sqlConnCat = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $catDB );
+
+if( $sqlConnCat->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->